packages/p2p/src/components/orders/orders.jsx

```diff
@@ -1,9 +1,9 @@
 import React        from 'react';
-import { localize } from 'deriv-translations';
 import PageReturn   from 'Components/page-return/page-return.jsx';
 import OrderDetails from './order-details/order-details.jsx';
 import OrderInfo    from './order-info';
-import './orders.scss';
+import                   './orders.scss';
+import { localize } from '../i18next';
```

Comment:
Please switch places. The ones that are simple `import '..path'` should be after all the `import..from..`s.